articles/install-command-line-qdk.md

```diff
@@ -52,7 +52,8 @@ Jupyter Notebooks allows running code in-place alongside instructions, notes, an
 While you can build Q# applications in any IDE, we recommend using Visual Studio Code (VS Code) or Visual Studio IDE for developing your Q# applications if you are running them via a .NET console. Developing in these environments leverages the rich functionality of the Quantum Development Kit (QDK) extension, which includes warnings, syntax highlighting, project templates, and more.
 
 > [!IMPORTANT]
-> If you are working on Linux, you may encounter a missing dependency depending on your particular distribution and installation method (such as certain Docker images). Please make sure that the `libgomp` library is installed on your system, as the GNU OpenMP support library is required by the quantum simulator of the QDK. On Ubuntu, you can do so by running `sudo apt install libgomp1`, or `yum install libgomp` on CentOS. For other distributions, please refer to your particular package manager.
+> If you are working on Linux, you may encounter a missing dependency depending on your particular distribution and installation method (such as certain Docker images). Please make sure that the `libomp` library is installed on your system, as the OpenMP support library is required by the quantum simulator of the QDK. On Ubuntu, you can do so by running `sudo apt install libomp5-11`, or `yum install libomp` on CentOS. For other distributions, please refer to your particular package manager.  
+On MacOS you can install `libomp` by running `brew install libomp`.
```

Comment:
> Is it OK to assume that macOS users have `brew` available?

I'm not very experienced in macOS but I would expect that "Yes", since anything that gets installed on macOS is mostly installed with `brew` (same as on some distros of Linux everything is mostly installed with `apt`).

> Apple uses the capitalization "macOS."

Fixing...

Reply:
I guess the difference is that `apt` is an official part of most Linux distributions, while `brew` is a very popular third-party tool for macOS. I have no objection to listing `brew`, but we may want to give context around what it is, or at least to have fallback for users without `brew`. By analogy, we give `conda`-based instructions as well as `pip`+`dotnet` instructions as a fallback.

Reply:
To move forward I plan to leave the change as is for now. Feel free to provide a desired change and I'll update this PR.

Reply:
@cgranade Do you want to suggest an edit to add a sentence?